### DISCUSSION TITLE
Saving a clip..

### DISCUSSION THREAD

**Discussion by jvrobert** [blakeblackshear/frigate#8977] [Ask A Question] [answered]
Is there a way to explicitly save a clip? Would be nice if there was, say, a lock icon on the clip playback interface and if you click it it excludes that clip from being deleted.

**Comment by NickM-27** [comment, marked as answer]
yes, the favorite button on the top right 
<img width="1512" alt="Screenshot 2023-12-15 at 2 35 49 PM" src="https://github.com/blakeblackshear/frigate/assets/14866235/42978900-14a5-4032-bf4b-9d07079ba64e">

**Reply by jvrobert** [reply]
Oh, duh, I should have noticed that ;)  Thanks!
